### DISCUSSION TITLE
Choose non-schematic placed blocks to ignore

### DISCUSSION THREAD

**Discussion by 6uis6b** [maruohon/litematica#684] [Ideas]
Is it possbile to add a function same as "ignoreExistingFluids" but with custom list? 
Last time i wanted to build with litematica i got annoyed that half of my schematic was red because of snowfall. So it would be nice if there was an option at least to ignore some specific things, but, if possible, anything you want

**Comment by maruohon** [comment]
Better rendering ignore lists are planned at some point. But specifically for snow layers and similar "small obstructions" you can enable the `renderCollidingSchematicBlocks` option (which is currently still also required as the second option with `ignoreExistingFluids` for the proper effect) to allow the schematic blocks to render even if there is already a block in that position in the world. The red overlay will be there, but at least you can see which block is supposed to go there (assuming it's larger than the obstructing block...).

**Reply by 6uis6b** [reply]
I already enabled it (accidentally found before I needed it) but thanks for your reply and good luck with future updates!
